assemblies/assembly-configuring-high-availability.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: HighAvailability
+[id="{context}"]
+= Configuring High Availability in {product}
+
```

Comment:
Consider starting the introduction by describing the default behavior, and explaining the HA banefits (why, for production, a customer should enable HA).